developersample/src/main/java/com/microsoft/identity/client/developersample/AuthUtil.java

```diff
@@ -0,0 +1,215 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.client.developersample;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.util.Log;
+
+import com.microsoft.identity.client.AuthenticationCallback;
+import com.microsoft.identity.client.AuthenticationResult;
+import com.microsoft.identity.client.ILoggerCallback;
+import com.microsoft.identity.client.Logger;
+import com.microsoft.identity.client.MsalClientException;
+import com.microsoft.identity.client.MsalException;
+import com.microsoft.identity.client.MsalServiceException;
+import com.microsoft.identity.client.MsalUiRequiredException;
+import com.microsoft.identity.client.PublicClientApplication;
+import com.microsoft.identity.client.UiBehavior;
+import com.microsoft.identity.client.User;
+
+import java.util.List;
+
+final class AuthUtil {
+
+    private static final String TAG = AuthUtil.class.getSimpleName();
+    private static final Object LOCK = new Object();
+    // There should be only one instance of PublicClientApplication per app
+    private static PublicClientApplication mApplication = null;
+    private static final String[] SCOPES = {"https://graph.microsoft.com/User.Read"};
+    private static final String[] EXTRA_SCOPES = {"Calendars.Read"};
+    private static final String CLIENT_ID = "9851987a-55e5-46e2-8d70-75f8dc060f21";
+    private List<User> mUsers = null;
```

Comment:
Seems like you could initialize an empty `List` here and save yourself a `null` check in `getUserCount()` -- this is minor but it jumped out at me